File: testserver/src/main/java/de/fluxparticle/animation/servlet/AnimationTestServer.java
package de.fluxparticle.animation.servlet;

import com.google.inject.servlet.GuiceFilter;
import org.eclipse.jetty.server.Server;
import org.eclipse.jetty.servlet.ServletContextHandler;

import java.net.URL;
import java.net.URLClassLoader;
import java.util.EnumSet;

import static javax.servlet.DispatcherType.REQUEST;

/**
 * Created by sreinck on 26.04.18.
 */
public class AnimationTestServer {

    public static void main(String[] args) throws Exception {
        URLClassLoader classLoader = (URLClassLoader) ClassLoader.getSystemClassLoader();
        for (URL url : classLoader.getURLs()) {
            System.out.println(url);
        }

        ServletContextHandler context = new ServletContextHandler();
        context.addEventListener(new AnimationServletTestConfig());
        context.addFilter(GuiceFilter.class, "/*", EnumSet.of(REQUEST));

        Server server = new Server(8080);
        server.setHandler(context);
        server.start();
        server.join();
    }

}

File: testserver/src/main/kotlin/de/fluxparticle/animation/servlet/AnimationTestServer.kt
package de.fluxparticle.animation.servlet

import com.google.inject.servlet.GuiceFilter
import org.eclipse.jetty.server.Server
import org.eclipse.jetty.servlet.ServletContextHandler
import java.net.URLClassLoader
import java.util.*
import javax.servlet.DispatcherType
import javax.servlet.DispatcherType.REQUEST

/**
 * Created by sreinck on 26.04.18.
 */
fun main(args: Array<String>) {
    val classLoader = ClassLoader.getSystemClassLoader() as URLClassLoader
    classLoader.urLs
            .map { it.toString() }
            .filter { it.contains("/Animation/") }
            .forEach { url ->
        println(url)
    }

    val context = ServletContextHandler()
    context.addEventListener(AnimationServletTestConfig())
    context.addFilter(GuiceFilter::class.java, "/*", EnumSet.of<DispatcherType>(REQUEST))

    val server = Server(8080)
    server.handler = context
    server.start()
    server.join()
}
